src/parser/parsed_data/copy_info.cpp

```diff
@@ -0,0 +1,29 @@
+#include "duckdb/parser/parsed_data/copy_info.hpp"
+
+namespace duckdb {
+
+unique_ptr<CopyInfo> CopyInfo::Copy() const {
+	auto result = make_uniq<CopyInfo>();
+	result->catalog = catalog;
+	result->schema = schema;
+	result->table = table;
+	result->select_list = select_list;
+	result->file_path = file_path;
+	result->is_from = is_from;
+	result->format = format;
+	result->options = options;
+	return result;
+}
+
+string CopyInfo::ToString() const {
+	string result = "";
+	if (is_from) {
+		throw NotImplementedException("TODO COPYINFO::TOSTRING");
```

Comment:
Heh looks like I left a TODO in here still, I'll fix that up

Reply:
This was actually implemented, but it lived in CopyStatement
It took all of its data out of CopyInfo regardless, except for `unique_ptr<QueryNode> select_statement;` which I've moved into CopyInfo.

This does have impact on serialization
CopyStatement doesn't have a (De)Serialize method yet, but CopyInfo does, so we might break forwards compatibility here (though I don't believe we serialize CopyInfo anywhere?)